risc0/zkvm/lib.rs

```diff
@@ -23,12 +22,16 @@ pub mod ffi {
     unsafe extern "C++" {
         include!("risc0/zkvm/prove/method_id.h");
 
-        #[cxx_name = "makeMethodId"]
-        fn make_method_id_from_elf(path: &CxxString) -> Result<[u8; 384]>;
+	// Unfortunately cxx::bridge doesn't seem to let us reference METHOD_ID_LEN here. :(
+        unsafe fn methodIdBytesFromElf(elf_contents: *const u8, len: usize) -> Result<[u8; 384]>;
```

Comment:
In #167 I'm using a `Vec<u8>` (to support dynamically sized MethodIDs) and using a cross-language FFI.

In the future, once the rust prover port lands, I suspect most of this FFI stuff will go away.